### PR TITLE
Batch working group meeting details

### PR DESCRIPTION
**Pull request by ahg-g**
Batch WG meeting details as agreed in https://groups.google.com/a/kubernetes.io/g/wg-batch/c/PoOqo6kSWWE

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -2930,16 +2930,22 @@ workinggroups:
       company: Intel
   meetings:
   - description: Regular Meeting
```
Comment (alculquicondor) [inline sigs.yaml:2932]: nit: put the calendar in both descriptions

File: sig-docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Contributing to SIG Docs
```
Comment (alculquicondor) [inline sig-docs/CONTRIBUTING.md:1]: where did this come from?

File: sigs.yaml (hunk 1 above)
Comment (ahg-g) [reply sigs.yaml]: yup, fixed.

File: sig-docs/CONTRIBUTING.md (hunk 2 above)
Comment (ahg-g) [reply sig-docs/CONTRIBUTING.md]: not sure, reverted

**Comment by ahg-g** [comment]
something is off, give me a second

**Pull request closed without merging**